content/docs/command-reference/config.md

```diff
@@ -238,6 +238,20 @@ to learn more about the state file (database) that is used for optimization.
   When an entry in the database is used (e.g. during the `dvc status`), DVC
   updates the timestamp on that entry. This way, when the database needs a
   cleanup, DVC can sort entries chronologically, and remove the oldest ones.
+  
+- `state.dir` - specify a custom location for the state file. This may be
+  necessary when using DVC on NFS or other mounted volumes.
+  ⚠️ Make sure no other project shares the same `state.dir` location.
```

Comment:
Sorry, still confused. Which state file does this refer to? https://dvc.org/doc/user-guide/project-structure/internal-files doesn't use that term and the note introduced in #2826 doesn't clarify this either. Does this affect .dvc/tmp/md5s or .dvc/tmp/links ? Or both

Reply:
I tried to document it more clearly: state.dir affects the `links` and `md5s` directories, index.dir the `index` directory.